go/cmd/berty/magisearch.go

```diff
@@ -0,0 +1,95 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+
+	ff "github.com/peterbourgon/ff/v3"
+	"github.com/peterbourgon/ff/v3/ffcli"
+	"go.uber.org/zap"
+
+	"berty.tech/berty/v2/go/internal/config"
+	"berty.tech/berty/v2/go/pkg/magisearch"
+	magiEngineBerty "berty.tech/berty/v2/go/pkg/magisearch/engines/berty"
+	magiParseBerty "berty.tech/berty/v2/go/pkg/magisearch/parsers/berty"
+	magiProvManager "berty.tech/berty/v2/go/pkg/magisearch/providers/manager"
+	magiProvRdvp "berty.tech/berty/v2/go/pkg/magisearch/providers/rdvp"
+)
+
+func magisearchCommand() *ffcli.Command {
+	fsBuilder := func() (*flag.FlagSet, error) {
+		fs := flag.NewFlagSet("berty magisearch", flag.ExitOnError)
+		manager.SetupLocalMessengerServerFlags(fs) // we want to configure a local messenger server
+		manager.SetupDefaultGRPCListenersFlags(fs)
+		return fs, nil
+	}
+
+	return &ffcli.Command{
+		Name:           "magisearch",
```

Comment:
```suggestion
		Name:           "omnisearch",
```

omnisearch is the official name of similar features on most other solutions

Reply:
```suggestion
		Name:           "search",
```

even better for the usage, feel free to use the term `omnisearch` as explanation, package name, but the CLI usage should be `berty search QUERY`